extensions/PhabBugz/lib/Util.pm

```diff
@@ -80,6 +80,22 @@ sub set_attachment_approval_flags {
       . $approval_flag_name
       . $status);
 
+  # If we are setting to + or -, we need to make sure
+  # Phabricator user is a member of the release-managers project
+  my $is_release_manager = 0;
+  my $release_manager_project
+    = Bugzilla::Extension::PhabBugz::Project->new_from_query(
+    {name => 'release-managers'});
+  if ($release_manager_project) {
```

Comment:
Nit: maybe if we don't find the `#release-managers` group that should be a fatal error? Seems we might not find it and then we will continue on.

Reply:
In that case $is_release_manager would still be 0. This would block setting of + or - for approval flags later so would just never set those statuses.